openhands/server/session/session.py

```diff
@@ -139,6 +139,8 @@ async def on_event(self, event: Event):
             return
         if event.source == EventSource.AGENT:
             await self.send(event_to_dict(event))
+        elif event.source == EventSource.USER and isinstance(event, CmdOutputObservation):
```

Comment:
Why does this observation have event source USER, below it's ENV ?

Reply:
I thought it was because the original action came directly from the user? (The user typed in the command)